ops/services/okta-global/main.tf

```diff
@@ -30,17 +30,18 @@ resource "okta_auth_server_claim" "given_name" {
 resource "okta_group" "prime_users" {
   name        = "Prime Team Members"
   description = "All Prime team members"
+  skip_users  = true
 }
 
 // Create a sign on policy requiring MFA
-
 resource "okta_policy_signon" "mfa_require" {
-  name            = "simple-report-mfa-require"
+  name            = "MFA policy"
```

Comment:
Are we going to have any issues with a space in the name? I would prefer to make this lowercase with a hyphen, just to keep it stylistically consistent.

Also, confirming...why are we changing the name, here?

Reply:
Any change in the okta-global has been made to match was is set in our resources, I have leaned into doing that as opposed to changing resources actively being used. This resource in Okta is set to `MFA policy`, not `simple-report-mfa-require`.